src/components/KFileUpload/KFileUpload.vue

```diff
@@ -298,43 +313,43 @@ const resetInput = (): void => {
   }
 
   .remove-button {
-    background-color: transparent;
+    $kFileUploadRemoveButtonPadding: var(--kui-space-30, $kui-space-30);
+    background-color: var(--kui-color-background-transparent, $kui-color-background-transparent);
     border: none;
     cursor: pointer;
     height: var(--spacing-lg);
-    padding: var(--type-xxs) 6px;
+    padding: $kFileUploadRemoveButtonPadding;
     position: absolute;
-    right: 118px;
-    top: 38px;
+    right: $tmp-spacing-120;
```

Comment:
issue: this is named too similarly to a kui token

question: why is this a temp variable? Seems like it would just remain hard-coded?

Reply:
Removed this, left it there accidentally